resources/quality/ultimaker_methodx/um_methodx_1c_um-abscf-175_0.2mm.inst.cfg

```diff
@@ -12,31 +12,52 @@ variant = 1C
 weight = -2
 
 [values]
-cool_fan_enabled = False
-raft_airgap = 0.3
+cool_fan_enabled = =True if extruder_nr == support_extruder_nr else False
+cool_fan_full_layer = 3
+cool_fan_speed = 0
+cool_fan_speed_0 = 0
+cool_fan_speed_max = 100
+cool_min_layer_time = 6
+cool_min_layer_time_fan_speed_max = 11
+cool_min_speed = 6
+cool_min_temperature = 250
+raft_airgap = =0.15 if extruder_nr == support_extruder_nr else 0
+raft_base_wall_count = =2 if extruder_nr == support_extruder_nr else 1
+raft_surface_flow = =110 if extruder_nr == support_extruder_nr else 100
+raft_surface_speed = =90 if extruder_nr == support_extruder_nr else 50
+raft_surface_thickness = =0.25 if extruder_nr == support_extruder_nr else 0.2
+raft_surface_z_offset = =-0.075 if extruder_nr == support_extruder_nr else 0
+skin_overlap = 10
+speed_layer_0 = 35
 speed_prime_tower = 30.0
 speed_print = 120.0
 speed_roofing = 55
-speed_topbottom = 55
+speed_support = =speed_print
+speed_topbottom = 85
 speed_travel = 250.0
 speed_wall_0 = 45
 speed_wall_x = 65
-support_angle = 50
+support_angle = 52
 support_bottom_density = 24
 support_bottom_enable = False
 support_bottom_line_width = 0.6
 support_bottom_stair_step_height = 0
 support_fan_enable = False
-support_infill_rate = 12.0
+support_infill_angles = [ 45,45,45,45,45,45,45,45,45,45,45,45,135,135,135,135,135,135,135,135,135,135,135,135]
```

Comment:
I'll trust you on this :smile:

Reply:
Haha, yes. It mimics the CloudPrint thatched linear fill.